app/serializers/api/v1/participant_from_induction_record_serializer.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require "jsonapi/serializer/instrumentation"
+
+module Api
+  module V1
+    class ParticipantFromInductionRecordSerializer
+      include JSONAPI::Serializer
+      include JSONAPI::Serializer::Instrumentation
+
+      class << self
+        def active_participant_attribute(attr, &blk)
+          attribute attr do |induction_record, params|
+            if induction_record.participant_profile.active_record?
+              if blk.parameters.count == 1
+                blk.call(induction_record)
+              else
+                blk.call(induction_record, params)
+              end
+            end
+          end
+        end
+
+        def trn(profile)
+          profile.teacher_profile.trn || profile.ecf_participant_validation_data&.trn
+        end
+
+        def validated_trn(profile)
+          eligibility = profile.ecf_participant_eligibility
+          eligibility.present? && !eligibility.different_trn_reason?
+        end
+
+        def eligible_for_funding?(profile)
+          ecf_participant_eligibility = profile.ecf_participant_eligibility
+          return if ecf_participant_eligibility.nil?
+          return true if ecf_participant_eligibility.eligible_status?
+          return false if ecf_participant_eligibility.ineligible_status?
+        end
+      end
+
+      set_type :participant
+
+      set_id :id do |induction_record|
+        # NOTE: using this will retain the original ID exposed to provider
+        induction_record.participant_profile.participant_identity.external_identifier
```

Comment:
unfortunately i think the ship has already sailed on that. we don't want to keep switching these back and forth it will just cause integration nightmares. so i'm leaving the existing implementation in